### PR TITLE
Rev-76 update the header menu

### PR DESCRIPTION
**Pull request by IbrahimAljabrST**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-76

### Description

-update the menu items font weight to be 600
-removed icons before the menu items

### Dependencies in Other PRs,Repo

none

### Testing Instructions

-check if the menu items font weight is 600
-heck if the icons before the menu items is removed

### CODE REVIEW

File: src/components/shared/Navbar/navbar.styles.js [hunk 1]
```diff
@@ -230,6 +226,7 @@ export const LogoutWrapper = styled.button`
   cursor: pointer;
   display: flex;
   font-size: 0.875rem;
+  font-weight: 600;
```
Comment (mohdabed123) [inline src/components/shared/Navbar/navbar.styles.js:229]:
```suggestion
  font-weight: bold;
```

File: src/components/shared/Navbar/navbar.styles.js [hunk 2]
```diff
@@ -203,12 +203,8 @@ export const MenuItem = styled.div`
   align-items: center;
   cursor: pointer;
   display: flex;
-  margin: 0.5rem 0;
-
-  svg {
-    margin: 0 0.5rem;
-    width: 1rem;
-  }
+  font-weight: 600;
```
Comment (mohdabed123) [inline src/components/shared/Navbar/navbar.styles.js:206]:
```suggestion
  font-weight: bold;
```

**Review by mohdabed123 — APPROVED** [review]
small changes other than that LGTM

File: src/components/shared/Navbar/navbar.styles.js (hunk 1 above)
Comment (IbrahimAljabrST) [reply src/components/shared/Navbar/navbar.styles.js:229]: QA want it 600

File: src/components/shared/Navbar/navbar.styles.js (hunk 2 above)
Comment (IbrahimAljabrST) [reply src/components/shared/Navbar/navbar.styles.js:206]: QA want it 600

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**